src/mission.cpp

```diff
@@ -2200,6 +2200,11 @@ static bool _intAddMissionResult(bool result, bool bPlaySuccess, bool showBackDr
 		screen_RestartBackDrop();
 	}
 
+	// ensure these widgets are deleted before attempting to create
+	widgDelete(psWScreen, IDMISSIONRES_TITLE);
+	widgDelete(psWScreen, IDMISSIONRES_FORM);
+	widgDelete(psWScreen, IDMISSIONRES_BACKFORM);
```

Comment:
Probably could move these to a function and call it here and in `intRemoveMissionResultNoAnim()`.